x-pack/plugin/searchable-snapshots/src/test/java/org/elasticsearch/xpack/searchablesnapshots/cache/shared/FrozenCacheServiceTests.java

```diff
@@ -210,7 +210,9 @@ public void testDecay() throws IOException {
     }
 
     public void testCacheSizeRejectedOnNonFrozenNodes() {
-        String cacheSize = randomBoolean() ? new ByteSizeValue(size(500)).getStringRep() : new RatioValue(between(1, 100)).toString();
+        String cacheSize = randomBoolean()
+            ? new ByteSizeValue(size(500)).getStringRep()
+            : RatioValue.formatNoTrailingZerosPercent(new RatioValue(between(1, 100)).getAsPercent()).toString();
```

Comment:
Think this is calling `String#toString()`?

```suggestion
            : RatioValue.formatNoTrailingZerosPercent(new RatioValue(between(1, 100)).getAsPercent());
```

Reply:
Resolved with the new instance method I made. You will see soon the new commits after I incorporate your feedback.